examples/webgl_postprocessing_pixel.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>three.js webgl - postprocessing - pixel shader</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+        <style>
+            body {
+                background-color: #000000;
+                margin: 0px;
+                overflow: hidden;
+                font-family:Monospace;
+                font-size:13px;
+                /*text-align:center;*/
+                font-weight: lighter;
+            }
+
+            a {
+                color: gray;
+                font-weight: bold;
+            }
+
+            #info {
+                color: gray;
+                position: absolute;
+                bottom: 0px;
+                width: 100%;
+                padding: 15px;
+            }
+            
+            .dg.ac {
+                z-index: 1 !important;
+            }
+        </style>
+    </head>
+    <body>
+
+        <script src="../build/three.js"></script>
+
+        <script src="js/shaders/CopyShader.js"></script>
+        <script src="js/shaders/PixelShader.js"></script>
+        <script src="js/postprocessing/EffectComposer.js"></script>
+        <script src="js/postprocessing/RenderPass.js"></script>
+        <script src="js/postprocessing/ShaderPass.js"></script>
+        <script src="js/controls/TrackballControls.js"></script>
+        <script src='js/libs/dat.gui.min.js'></script>
+
+        <div id="container"></div>
+
+        <div id="info">
+            <a href="http://threejs.org" target="_blank" rel="noopener">three.js</a> - pixel shader example<br/>
+            shader by <a href="http://wongbryan.github.io">wongbryan</a>
+        </div>
+
+        <script>
+
+            var camera, scene, renderer, gui, composer;
+            var pixelPass, params;
+
+            var geomData = [], group;
+
+            init();
+            animate();
+
+            function updateGUI() {
+
+                pixelPass.uniforms.pixelSize.value = params.pixelSize;
+
+            }
+
+            function resize() {
+
+                camera.aspect = window.innerWidth / window.innerHeight;
+                camera.updateProjectionMatrix();
+                renderer.setSize( window.innerWidth, window.innerHeight );
```

Comment:
You will have to reset the resolution, too.
```javascript
pixelPass.uniforms.resolution.value.set( window.innerWidth, window.innerHeight ).multiplyScalar( window.devicePixelRatio );
```

Reply:
Updated the shader and resize function